.github/workflows/rust.yml

```diff
@@ -0,0 +1,23 @@
+name: Rust
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build
+      run: cargo build --verbose --manifest-path=rs/Cargo.toml
```

Comment:
Did you run it with errors in .rs files? Does it fail the PR in this case?

Reply:
Let me try that now and if so then I will complete the PR

Reply:
When I added a syntax error in a rust file the action broke: https://github.com/microsoft/dev-tunnels/actions/runs/5073570643/jobs/9112748431?pr=241 I will revert this error now since it works